src/components/symptom-shark/presentational/MonthReport/MonthReport.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useContext } from "react";
+import "./MonthReport.css";
+import NotesTimeline from "../NotesTimeline/NotesTimeline";
+import BulletChart from "../BulletChart/BulletChart";
+import OverallExperienceChart from "../OverallExperienceChart/OverallExperienceChart";
+import SymptomSeverityChart from "../SymptomSeverityChart/SymptomSeverityChart";
+import { getMonthName } from "../../../../helpers/date-helpers";
+import { DateRangeContext, TextBlock } from "../../../presentational";
+import { SymptomSharkVisualizationContext } from "../../container/VisualizationCoordinator/VisualizationCoordinator";
+import { startOfMonth } from "date-fns";
+
+export interface SymptomSharkMonthReportProps {
+    includeNotes: boolean;
+    includeDailyOverallFeeling: boolean;
+    productLogo?: string;
+    productName?: string;
+    productUrl?: string;
+    intervalStart?: Date;
+    innerRef?: React.Ref<HTMLTableElement>;
+}
+
+export default function (props: SymptomSharkMonthReportProps) {
+    let visualizationContext = useContext(SymptomSharkVisualizationContext);
+    if (!visualizationContext) {
+        return <TextBlock innerRef={props.innerRef}>Error: Month report must be used inside a Symptom Shark Visualization Coordinator.</TextBlock>
+    }
+
+    let dateRangeContext = useContext(DateRangeContext);
+    let intervalStart = props.intervalStart || startOfMonth(new Date());
+    if (dateRangeContext) {
+        intervalStart = dateRangeContext.intervalStart;
+    }
+
+    if (!visualizationContext.symptoms.length && !visualizationContext.treatments.length) {
+        return null;
+    }
+
+    return <table ref={props.innerRef} className="ss-month-report" cellPadding="0" cellSpacing="0">
+        <thead>
+            <tr>
+                <th>
+                    <div className="header">
+                        {getMonthName(intervalStart.getMonth())} {intervalStart.getFullYear()}
+                        <div className="product">
+                            {props.productLogo &&
+                                <img src={props.productLogo} />
+                            }
+                            {props.productName &&
+                                <div className="product-name">{props.productName}</div>
+                            }
+                            {props.productUrl &&
+                                <a href={"https://" + props.productUrl} className="product-url">{props.productUrl}</a>
```

Comment:
Why at the prefix here?  Shouldn't we just have folks include that as part of the URL property value?

Reply:
Removed.  No idea why I did this originally